sdgym/benchmark.py

```diff
@@ -80,18 +79,15 @@ def _generate_job_args_list(limit_dataset_size, sdv_datasets, additional_dataset
     max_rows, max_columns = (1000, 10) if limit_dataset_size else (None, None)
     run_id = os.getenv('RUN_ID') or str(uuid.uuid4())[:10]
 
-    synthesizers = get_synthesizers(synthesizers)
-    if custom_synthesizers:
-        custom_synthesizers = get_synthesizers(custom_synthesizers)
-        synthesizers.extend(custom_synthesizers)
-
-    datasets = []
-    if sdv_datasets is not None:
-        datasets = get_dataset_paths(sdv_datasets, None, None, None, None)
+    # Get list of synthesizer objects
+    custom_synthesizers = [] if custom_synthesizers is None else custom_synthesizers
```

Comment:
small nit pick you can leave it as it is if you like it more, but usually:
```python
custom_synthesizers = custom_synthesizers if custom_synthesizers else []
```
In this case its not a huge difference but in the following ones you see what method is going to be called or what they should be if that is true, otherwise a quick glance tells you that this is empty list.